api/src/controllers/utils.ts

```diff
@@ -207,8 +189,8 @@ export const getTimespanRange = (
       return [
         {
           range: {
-            'filter.times.startDateTime': {
-              gte: 'now',
+            'filter.times.endDateTime': {
```

Comment:
this change seems like it's deliberate, but just checking it's definitely what we want?

Reply:
Yes! So it says as long as the **end** date is greater than now, consider it the future, because it's not ended yet.

The previous query said "if the start date if greater or equal to now it's future", but an exhibition that started the previous month would then not have been returned.